src/test/ui/where-clauses/higher-ranked-fn-type.quiet.stderr

```diff
@@ -0,0 +1,18 @@
+error[E0277]: the trait bound `for<'b> for<'b> fn(&'b ()): Foo` is not satisfied
```

Comment:
The diagnostic looks a bit silly. I don't know why we are choosing a `'b` for the inner param.